argus/action_manager/windows.py

```diff
@@ -306,7 +306,7 @@ def mkdir(self, path):
         """
         if self.exists(path):
             raise exceptions.ArgusCLIError(
-                "Cannot create directory ‘{}’ The path already exists".format(
+                "Cannot create directory '{}'. The path already exists".format(
```

Comment:
I think it would be more appropriate to log `Cannot create directory {} . It already exists.`